cinn/runtime/buffer.cc

```diff
@@ -26,7 +26,6 @@ Shape::Shape(const Shape &other) : data_(new value_type[other.ndims()]), ndims_(
 void Shape::Resize(int ndim) {
   CHECK_GT(ndim, 0);
   ndims_ = ndim;
-  if (data_) delete data_;
```

Comment:
:thought_balloon: I find the deletion of such a statement questionable.

Reply:
I agree with it. @enkilee `if (data_) delete data_;`->`delete data_;`

Reply:
OK